### DISCUSSION TITLE
No Destination found for a deployment for VMinstance

### DISCUSSION THREAD

**Discussion by chinmaya-kumar-biswal** [apache/cloudstack#9372] [Q&A] [answered]
#Create VM Instance
While create a VM it was showing No Destination found for a deployment for VM instance

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by DaanHoogland** [comment, marked as answer]
@chinmaya-kumar-biswal there are so many reasons this could happen. The one that comes to mind is that the requirements of the VM is 3 CPU while the Hosts in your env have 2 at most. No primary storage reachable is another one. We really need more information (error messages, logs) to be able to help you.
